phpBB/styles/prosilver/template/attachment.html

```diff
@@ -29,6 +29,26 @@
 		</dl>
 		<!-- ENDIF -->
 
+		{% if _file.S_AUDIO_FILE %}
+			<dl class="file">
+				<dt class="c-file-header">
+					<audio class="c-file-audio-controls" controls="controls" preload="metadata">
+						<source class="c-file-audio-source" src="{{ _file.U_DOWNLOAD_LINK }}" type="{{ _file.MIMETYPE }}">
+					</audio>
+				</dt>
+				{% if _file.COMMENT %}
+					<dd class="c-file-comment">{{ _file.COMMENT }}</dd>
+				{% endif %}
+				<dd class="c-file-actions">
+					{% if _file.UPLOAD_ICON %}
+						<span class="o-icon c-file-icon">{{ _file.UPLOAD_ICON }}</span>
```

Comment:
again not sure where this icon is coming from...does it need to be a DB icon? Is it user specified in some way? Eitherway it needs an svg implementation using the icon twig function The goal is all icon/image assets in 4.0 use this to include eventually smiles and rank imgs. May be outside of the scope here but if I do not ask I will forget down the road...

Reply:
This is from the database, correct. I think we have similar things to change for the topic icons and should probably also rethink these types of icons then. I do also agree that it is outside the scope of this PR.